wp-content/themes/cpipr/partials/content-series.php

```diff
@@ -47,7 +52,15 @@
 
 			<div class="title-article">
 				<h2><?php the_title(); ?></h2>
-				<p class="date-text"><?php the_time('j F, Y') ?></p>
+				<p class="date-text">
+					<?php
+						if (has_category('english', get_the_ID())) {
+							echo date('j F Y', get_the_time( 'U', get_the_ID() ));
+						} else {
+							the_time('j F Y');
+						}
+					?>
+				</p>
```

Comment:
Hi @benlk, 
We use a date function instead of largo_time so we make sure that the date renders in the correct language, in this case Spanish,  on a template by template basis.